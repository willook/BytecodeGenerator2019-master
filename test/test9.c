int main() {
    int j = 0;
    int i = 0;
    int num[5];
    num[i] = 9;
    num[++i] = 8;
    num[3] = i + 1;
    num[2] = ++i;

    while (j < 5) {
        _print(num[j]);
        j = j + 1;
    }
}