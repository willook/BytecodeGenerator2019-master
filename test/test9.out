9
8
2
2
0